src/Ext/TechnoType/Body.h

```diff
@@ -43,11 +43,15 @@ class TechnoTypeExt
 		Valueable<bool> MultiMindControl_ReleaseVictim;
 		Valueable<int> CameoPriority;
 		Valueable<bool> NoManualMove;
+		Valueable<bool> AutoDeployOnDepletedAmmo;//testing
 		Nullable<int> InitialStrength;
 
 		Valueable<ShieldTypeClass*> ShieldType;
 		std::unique_ptr<PassengerDeletionTypeClass> PassengerDeletionType;
 
+		Valueable<bool> AmmoDepAction_AutoDeploy;
+		Valueable<bool> AmmoDepAction_DeployBlock;
```

Comment:
```suggestion
		Valueable<bool> OnAmmoDepletion_AutoDeploy;
		Valueable<bool> OnAmmoDepletion_DeployBlock;
```

Class members should have the same name as INI tags.